python/hail/matrixtable.py

```diff
@@ -635,6 +646,27 @@ def col(self):
         """
         return self._col
 
+    @property
+    def col_value(self):
+        """Returns a struct expression including all non-key column-indexed fields.
+
+        Examples
+        --------
+
+        Get all non-key column field names:
+
+        .. doctest::
```

Comment:
same two comments as above: extra whitespace after Example header, and don't need doctest directive